Podverse/Constants.swift

```diff
@@ -288,28 +288,37 @@ enum EpisodeActions: String {
     }
 }
 
-enum ErrorMessages: String {
-    case noClipsAvailable
-    case noClipsInternet
-    case noPlaylistsAvailable
-    case noPlaylistsInternet
-    case noPlaylistsNotLoggedIn
+struct Strings {
     
-    var text: String {
-        get {
-            switch self {
-            case .noClipsAvailable:
-                return "No clips available"
-            case .noClipsInternet:
-                return "No internet connection"
-            case .noPlaylistsAvailable:
-                return "No playlists available"
-            case .noPlaylistsInternet:
-                return "No internet connection"
-            case .noPlaylistsNotLoggedIn:
-                return "Login to view your playlists"
-            }
-        }
+    struct Errors {
+        static let noClipsAvailable = "No clips available"
```

Comment:
I don't think these need to be static. I believe just `let`s should work fine

Reply:
I get this error when I remove static:

![image](https://user-images.githubusercontent.com/2262919/31648524-dce8661e-b2d3-11e7-94fb-2f311d9272ee.png)

Reply:
Both you and the compiler are absolutely correct and I apologize. There is no initialization of the Strings struct so the properties need to be static. My bad

Reply:
Haha no problem.